Task1/script.js

```diff
@@ -0,0 +1,39 @@
+const th = table.querySelectorAll('th');
+const tbody = table.querySelector('tbody');
+const row = Array.from(tbody.rows);
+const input = document.createElement('input');
+
+[].forEach.call(th, function (th, elem) {
+  th.addEventListener('click', function () {
+    sort(elem);
+  });
+});
+
+tbody.addEventListener('click', cellClick)
+
+function sort(elem) {
+
+  let compare = function (rowA, rowB) {
+    return rowA.cells[elem].innerHTML > rowB.cells[elem].innerHTML ? 1 :
```

Comment:
your sorting work only in one direction. if I click again the list wont be sorted